include/vrv/vrvdef.h

```diff
@@ -672,6 +672,7 @@ enum GraphicID { PRIMARY = 0, SPANNING, SYMBOLREF };
 //----------------------------------------------------------------------------
 
 #define TABLATURE_STAFF_RATIO 1.75
+#define GERMAN_TAB_STAFF_RATIO 2.2
```

Comment:
Could you explain why you need a distinct ratio for German tablature?

Reply:
With the smaller ratio I found that the Gothic characters overlapped each other and were very cramped, the Gothic characters are quite bulky and some are very tall, especially the k.